arangodb/content.md

```diff
@@ -4,9 +4,7 @@ ArangoDB is a multi-model, open-source database with flexible data models for do
 
 The supported data models can be mixed in queries and allow ArangoDB to be the aggregation point for the data request you have in mind.
 
-> [arangodb.com](https://arangodb.com)
-
-%%LOGO%%
+Dockerfile: [`Latest` (Dockerfile)](https://github.com/arangodb/arangodb-docker/blob/master/Dockerfile.templ)
```

Comment:
I realize this is a sync from your other readme, but there are [links generated](https://github.com/docker-library/docs/blob/master/arangodb/README.md#supported-tags-and-respective-dockerfile-links) for each Dockerfile for each version.